components/avatar/avatar.vue

```diff
@@ -114,6 +129,25 @@ export default {
       type: String,
       default: '',
     },
+
+    /**
+     * Determines whether to show a gradient background for the avatar.
+     */
+    gradient: {
+      type: Boolean,
+      default: true,
+    },
+
+    /**
+     * Determines whether to show a group avatar.
+     * Limit to 2 digits max, though supports “99+”.
+     * if the number is 1 or less it would just show the regular avatar as if group had not been set.
+     */
+    group: {
+      type: [Number, String],
+      default: undefined,
+      validator: (group) => AVATAR_GROUP_VALIDATOR(group),
+    },
```

Comment:
the control input renders as an "object" in storybook I think due to the multiple types. Probably not even necessary to allow both types, maybe only allow "number"

Reply:
I think we can also just set it to 99+ when any number greater than 99 is entered.

Reply:
yeah followed the approach you've suggested, thanks https://github.com/dialpad/dialtone-vue/pull/745/commits/541fb7968a722db9b7ac493ec648f1f17f588c99